### PR TITLE
Enable docker-compose test

### PR DESCRIPTION
**Pull request by pavelkalinnikov**
Part of #2298.

### CODE REVIEW

File: examples/deployment/docker-compose.yml [hunk 1]
```diff
@@ -68,3 +68,10 @@ services:
     depends_on:
       - mysql
 
+# TODO(pavelkalinnikov): Find a way to remove this network declaration. It is
```
Comment (AlCutter) [inline examples/deployment/docker-compose.yml:71]: I wonder if putting the network specific bits in a separate `yml` file and using something like below might work:
```
docker-compose -d examples/deployment/docker-compose.yml -f integration/cloudbuild/network.override.yml up
```
?

**Review by AlCutter — APPROVED** [review]

File: examples/deployment/docker-compose.yml (hunk 1 above)
Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: What is the "-d" flag? Can't find it.

Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: Oh wait, that's the detach. But I still don't get how the 2 files a fed to the docker-compose.

Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: It seems like you can't combine 2 docker-compose files like that? The "free" parameters are for the service names to start from the the config file, not other docker-compose files.

Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: Ah, but you can supply multiple `-f` parameters apparently: https://docs.docker.com/compose/extends/

Comment (AlCutter) [reply examples/deployment/docker-compose.yml:71]: Sorry, typo - that was meant to be `-f` twice >_<

Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: Ok, I'll merge this PR for now and try to play with this suggestion.

Comment (pavelkalinnikov) [reply examples/deployment/docker-compose.yml:71]: #2350.

**Pull request merged**